### PR TITLE
Example for a plastic synapse

### PR DESCRIPTION
**Pull request by schmitts**

### CODE REVIEW

File: python/example/single_cell_stdp.py [hunk 1]
```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python3
```
Comment (noraabiakar) [inline python/example/single_cell_stdp.py:1]: Can we add the example to github actions? https://github.com/arbor-sim/arbor/blob/master/.github/workflows/basic.yml

**Review by noraabiakar — CHANGES_REQUESTED** [review]
Looks good, thanks! Just a small request.

File: python/example/single_cell_stdp.py (hunk 1 above)
Comment (schmitts) [reply python/example/single_cell_stdp.py:1]: done

File: python/example/single_cell_stdp.py [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+    reg_sched = arbor.regular_schedule(0.1)
+    handle_mem = sim.sample((0, 0), reg_sched)
+    handle_g = sim.sample((0, 1), reg_sched)
+    handle_apost = sim.sample((0, 2), reg_sched)
+    handle_apre = sim.sample((0, 3), reg_sched)
+    handle_weight_plastic = sim.sample((0, 4), reg_sched)
+
+    sim.run(tfinal=600)
+    mem, meta = sim.samples(handle_mem)[0]
```
Comment (noraabiakar) [inline python/example/single_cell_stdp.py:98]: I think this line can be removed?

**Review by noraabiakar — APPROVED** [review]
Looks good! There's just one line in the example that I think can be removed, I didn't notice it before.

File: python/example/single_cell_stdp.py (hunk 2 above)
Comment (schmitts) [reply python/example/single_cell_stdp.py]: ack

**Pull request merged**